test/system/030-run.bats

```diff
@@ -608,6 +608,16 @@ json-file | f
     # a subdir of a volume.
     run_podman run --rm --workdir /IamNotOntheImage -v $testdir/content:/IamNotOntheImage/foo $IMAGE cat foo
     is "$output" "$randomcontent" "cat random content"
+
+    # Make sure that running on a read-only rootfs works (#9230).
+    skip_if_rootless "image mounts are too hard to test rootless"
```

Comment:
Please don't use `skip` in the middle of a test; that has the result of marking the entire test as skipped, which will confuse someone looking at logs. Instead, could you do:
```bash
    if ! is_rootless; then
        ...
```

Reply:
Incidentally, a cheap way to test as rootless is `--rootfs /`. Unfortunately, that always fails in one of these two ways:
```console
$ ./bin/podman run --rm --rootfs / echo hi
Error: open `//.containerenv`: No such file or directory: OCI not found
```
or
```console
$ ./bin/podman run --rm --rootfs / echo hi
Error: mkdir `secrets`: Permission denied: OCI permission denied
```
(I don't know why it's never a consistent failure)

Reply:
Looks like you also have to check for remote:
```bash
    if ! is_rootless && ! is_remote; then
        ...
```
(reason: podman-remote doesn't handle `image mount`)

Reply:
Thanks! Updated, also to skip if remote.